editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -271,4 +271,155 @@ describe('actions', () => {
       })
     })
   })
+  describe('UNWRAP_GROUP_OR_VIEW', () => {
+    it(`Unwraps a group`, async () => {
+      const testCode = `
+				<div data-uid='aaa' style={{contain: 'layout', width: 300, height: 300}}>
```

Comment:
the indentation police is here, they are asking about these tabs in the sample code 👮‍♀️
<img width="859" alt="image" src="https://user-images.githubusercontent.com/2226774/232052983-d70cb74c-1e71-4df1-9e3c-5dde739cd833.png">

Reply:
I don't know what they are talking about 🙃